upgrade.md

```diff
@@ -488,6 +488,8 @@ The `defer` boolean property on the service provider which is/was used to indica
 
 By default, Laravel 5.8 uses PHPUnit 7. However, you may optionally upgrade to PHPUnit 8, which requires PHP >= 7.2. In addition, please read through the entire list of changes in [the PHPUnit 8 release announcement](https://phpunit.de/announcements/phpunit-8.html).
 
+#### `setUp` And `tearDown`
```

Comment:
To avoid further confustion I would put it above PHPUnit 8 subsection

Reply:
Good thinking. Done.